Map funtionality

Hi @brockfanning, 

i have tried to use the map option - but something went wrong as cou can see [here](https://wohnungnds.github.io/WK_Site/1-1-1/)

I am not sure if I understand the docs right:

I put my geojson file in the data repsoitroy [/geojson/maps](https://github.com/WohnungNDS/WK_Data/tree/develop/geojson/maps) and added the following code in my [config ](https://github.com/WohnungNDS/WK_Site/blob/develop/_config.yml) file

```
map_options:
  minZoom: 5
  maxZoom: 10
  tileURL: https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}
  tileOptions:  
    id: mapbox.light
    accessToken: pk.eyJ1IjoiY2hyaXN0b3BoNDcxMSIsImEiOiJjanp1cjdpbzQwMTFwM29tdzJ2ZTh3Ymo4In0.C7_7e3AFcMaF_QBg6MaT0Q
    attribution: <a href="https://www.mapbox.com">Mapbox</a> | <a href="http://www.bkg.bund.de">© GeoBasis-DE / BKG</a>
```

 
```

map_layers:
    subfolder: maps
    # The label to use in the "Download GeoJSON" button:
    label: indicator.map
    # The minimum zoom at which this layer should be visible.
    min_zoom: 0
    # The maximum zoom at which this layer should be visible.
    max_zoom: 20
    # Whether or not these boundaries should display statically on lower layers.
    staticBorders: false
```
But the map will not appear....
Do you have any idea?

@brockfanning I have tried different things:

the geojson in my data repository
```
map_layers:
  - geojson_file: https://github.com/WohnungNDS/WK_Data/geojson/maps/NI_Kurz_diss.geojson
    name_property: GEN
    id_property: AGS
    output_subfolder: maps
    filename_prefix: indicator
```_
Produces the following Error: 
yaml.parser.ParserError: while parsing a block collection
  in "config_data.yml", line 92, column 4
expected <block end>, but found '<block mapping start>'
  in "config_data.yml", line 93, column 5
##[error]Process completed with exit code 1.
```

```
map_layers:
 - geojson_file: https://wohnungnds.github.io/WK_Data/geojson/maps/NI_Kurz_diss.geojson
    name_property: GEN
    id_property: AGS
    output_subfolder: maps
    filename_prefix: indicator

produces the following error:

urllib.error.HTTPError: HTTP Error 404: Not Found
##[error]Process completed with exit code 1.
```


```
map_layers:
  - geojson_file: - geojson_file: https://christoph4711.github.io/open-sdg-site-starter/assets/maps/NI_Kurz_diss.geojson
    name_property: GEN
    id_property: AGS
    output_subfolder: maps
    filename_prefix: indicator
```_
```

no errors but also no map

It looks like the geojson file will not be found in the data repository....

but the geojson file seems to be valid as you can see [here](https://github.com/WohnungNDS/WK_Data/blob/develop/geojson/maps/NI_Kurz_diss.geojson)

--- a/docs/maps.md
+++ b/docs/maps.md
@@ -25,7 +25,29 @@ Each "feature" in the The GeoJSON must have the following in their `properties` 
 1. A property that holds the unique ID for the boundary (which must correspond to the GeoCode data column mentioned above)
 2. A property that holds the human-readable name of the boundary
 
-The specific keys for these properties are configured in the data repository.
+The specific keys for these properties are configured in the data repository. Here is an example of the relevant configuration settings for the data repository:
+
+```
+map_layers:
+  -
+    # This can be a local relative path or a remote URL.
+    geojson_file: path/to/my/regions.geojson
+    # This is the property in the GeoJSON that holds the name of the region.
+    name_property: rgn17nm
+    # This is the property in the GeoJSON that holds the "geocode" of the region.
+    id_property: rgn17cd
+    # This controls the subfolder that the GeoJSON file will be created in.
+    output_subfolder: regions
+    # This adds a prefix to all the GeoJSON filenames.
+    filename_prefix: indicator_
+  -
+    # Add any number of additional layers, such as one for counties.
+    geojson_file: path/to/my/counties.geojson
+    name_property: cntynm
+    id_property: cntycd
+    output_subfolder: counties
+    filename_prefix: indicator_
+```
 
 ## Site configuration: map_options and map_layers
 

